eq-author/src/components/CommentsPanel/index.js

```diff
@@ -229,6 +231,33 @@ const CommentsPanel = ({
   const [deleteComment] = useMutation(COMMENT_DELETE);
   const [updateComment] = useMutation(COMMENT_UPDATE);
 
+  const commentsSubscription = gql`
+    subscription CommentsUpdated($pageId: ID!) {
+      commentsUpdated(pageId: $pageId) {
+        id
+        comments {
+          id
+          commentText
+          user {
+            id
+            name
+            picture
+            email
+            displayName
+          }
+          createdTime
+          editedTime
+        }
+      }
+    }
+  `;
```

Comment:
This should be outside the component. It should probably be moved to its own file too.

This part can probably be moved into a fragment if it is used in multiple places.
```graphql
        id
        comments {
          id
          commentText
          user {
            id
            name
            picture
            email
            displayName
          }
          createdTime
          editedTime
        }
```